modules/@apostrophecms/ui/ui/apos/scss/mixins/_theme_mixins.scss

```diff
@@ -37,6 +37,7 @@ $boolean-padding: $spacing-double;
   // stylelint-disable
   --a-primary-button-hover: #{mix(black, $color, 10%)};
   --a-primary-button-active: #{mix(black, $color, 15%)};
+  --a-primary-child: #{mix(red, $color, 50%)};
```

Comment:
I don't understand the semantics of this naming.

Reply:
changed this to full fledged `--a-secondary` 🤞